src/test/java/pl/edu/zut/mad/schedule/MockData.kt

```diff
@@ -106,7 +116,7 @@ internal class MockData {
         private fun getLessonUiWithCancellation(): LessonUi {
             val date = getUiDate()
             return LessonUi(SUBJECT, TYPE, ROOM, TEACHER_UI, FACULTY_ABBREVIATION, FIELD_OF_STUDY,
```

Comment:
Looks like it needs builder

Reply:
It looks that I have to add some boilerplate to get builder. I searching for some libraries for generating this specially for Kotlin but I didn't find anyting.